docs/conf.py

```diff
@@ -87,8 +87,8 @@
 html_context = {
     'current_language': 'English',
     'languages': (('English', '/latest'), ('Chinese', '/cn/latest')),
-    'doxygen_mapping_file': '@DOXYGEN_MAPPING_FILE@',
-    'doxygen_snippet_root': '@OpenVINO_SOURCE_DIR@'
+    'doxygen_mapping_file': '',
+    'doxygen_snippet_root': '/home/kputnam/ov2'
```

Comment:
Isn't this hardcoded path here left by the mistake?

Reply:
Good catch. My shortcut caught up with me. I'll fix it.